data/tokens.json

```diff
@@ -1227,56 +1227,56 @@
         "type": "color"
       }
     },
-    "gtmds_action-item-margin-right": {
+    "gtmds_action-item_margin-right": {
       "value": "{gtmds_size-sm}",
       "type": "spacing"
     },
-    "gtmds_action-item-margin-bottom": {
+    "gtmds_action-item_margin-bottom": {
       "value": "{gtmds_size-sm}",
       "type": "spacing"
     },
     "gtmds_action-item_padding": {
       "value": "{gtmds_size-md}",
       "type": "spacing"
     },
-    "gtmds-actionItem-sizeWidth": {
+    "gtmds_action-item_size-width": {
       "value": "auto",
       "type": "sizing"
     },
-    "gtmds-actionItem-sizeHeight-default": {
+    "gtmds_action-item_size-height": {
       "value": "{gtmds_size-xl}+{gtmds_size-md}+{gtmds_size-sm}",
       "type": "sizing"
     },
-    "gtmds-actionItem-sizeHeight-description": {
+    "gtmds_action-item_size-height-description": {
       "value": "{gtmds_size-xl}*2+{gtmds_size-sm}*3",
       "type": "sizing"
     },
-    "gtmds_action-item-indent_size-base": {
-      "value": "20",
+    "gtmds_action-item_indent-base": {
+      "value": "{gtmds_size-sm}+{gtmds_size-lg}",
       "type": "spacing"
     },
-    "gtmds_action-item-indent_size0": {
+    "gtmds_action-item_indent-0": {
       "value": "{gtmds_size-md}",
       "type": "spacing"
     },
-    "gtmds_action-item-indent_size1": {
-      "value": "{gtmds_action-item-indent_size0}+{gtmds_action-item-indent_size-base}",
+    "gtmds_action-item_indent-1": {
+      "value": "{gtmds_action-item_indent-0}+{gtmds_action-item_indent-base}",
       "type": "spacing"
     },
-    "gtmds_action-item-indent_size2": {
-      "value": "{gtmds_action-item-indent_size0}+{gtmds_action-item-indent_size-base}*2",
+    "gtmds_action-item_indent-2": {
+      "value": "{gtmds_action-item_indent-0}+{gtmds_action-item_indent-base}*2",
       "type": "spacing"
     },
-    "gtmds_action-item-indent_size3": {
-      "value": "{gtmds_action-item-indent_size0}+{gtmds_action-item-indent_size-base}*3",
+    "gtmds_action-item_indent-3": {
+      "value": "{gtmds_action-item_indent-0}+{gtmds_action-item_indent-base}*3",
       "type": "spacing"
     },
-    "gtmds_action-item-indent_size4": {
-      "value": "{gtmds_action-item-indent_size0}+{gtmds_action-item-indent_size-base}*4",
+    "gtmds_action-item_indent-4": {
+      "value": "{gtmds_action-item_indent-0}+{gtmds_action-item_indent-base}*4",
       "type": "spacing"
     },
-    "gtmds_action-item-indent_size5": {
-      "value": "{gtmds_size-md}+{gtmds_action-item-indent_size-base}*5",
+    "gtmds_action-item_indent-5": {
+      "value": "{gtmds_size-md}+{gtmds_action-item_indent-base}*5",
```

Comment:
Is it possible to rename these tokens to use something like left padding on the expander icon instead of using the indent term, which might be a bit ambiguous? @tsnikolov what do you think?

Reply:
You can check the latest commit.